GitCommands/Git/GitVersion.cs

```diff
@@ -172,11 +173,7 @@ private static int Compare(GitVersion left, GitVersion right)
 
         public override string ToString()
         {
-            return Full
-                .Replace(".windows.0", "")
-                .Replace(".windows.1", "")
-                .Replace(".windows.2", "")
-                .Replace(".windows.3", "");
+            return Regex.Replace(Full, @"\.windows\.\d+", "");
```

Comment:
This cleans the suffix in display. "windows" could be changed too, is different in wsl and wine I suppose.
An alternative could be to just display the version like below, but that will clean -rc etc
```
            return Regex.Replace(Full, @"(\d+(\.\d+)+)", "$1");
```

Why not display this part of the version instead, no replace at all?

Reply:
@vbjay ?

Reply:
I am not trying to alter functionality here.  The replace is being used for

https://github.com/gitextensions/gitextensions/blob/78092cb9d74b3068803d8b03d986f573553f0a83/GitUI/Infrastructure/Telemetry/AppEnvironmentTelemetryInitializer.cs#L26

I happened to catch the brittle case of a potential running out of numbers and it not being replaced.

Reply:
There is no harm in Telemetry tracking the full name, especially if alternative Git implementations are used.
@RussKie ?

Reply:
![image](https://user-images.githubusercontent.com/6248932/204155316-dcb31652-4451-4ff5-a9b5-8d280734ed82.png)

Reply:
Ok. I'll switch it to just use full.

Reply:
> There is no harm in Telemetry tracking the full name, especially if alternative Git implementations are used.

I _personally_ see no issues with that.

Reply:
> a friendly nudge

Last comment from @vbjay was to change to remove the clean, not done.

Reply:
Updated.